govcd/api_vcd.go

```diff
@@ -89,7 +89,7 @@ func NewVCDClient(vcdEndpoint url.URL, insecure bool) *VCDClient {
 
 	return &VCDClient{
 		Client: Client{
-			APIVersion: "5.5",
+			APIVersion: "27.0",
```

Comment:
Is it possible to make this version dynamic or depending on a global variable?

Reply:
If this is the intention, then let's put the supported API version in a constant with a telling name.

Reply:
What is you suggestion for name?

Reply:
Something like `maximumSupportedVersion`.

Reply:
`APIVersion` name is correct here. It just means our client supports API v27.0 - it is _not_ minimum or maximum supported version. What we could do to be more clear is to add a comment like `APIVersion: "27.0", // vCD 8.20`